src/compiler/utilities.ts

```diff
@@ -2547,6 +2507,52 @@ namespace ts {
         }
     }
 
+    /* @internal */

```

Comment:
I'd rather we keep to a consistent series of values for operator precedence, so we don't in the future accidentally try to compare the result from `getBinaryOperatorPrecedence` to `getOperatorPrecedence`. As far as the parser is concerned, it only cares whether the new precedence is higher than the old one and is greater than `0`.